pxteditor/monaco-fields/field_sprite.ts

```diff
@@ -56,7 +56,8 @@ namespace pxt.editor {
             return {
                 initWidth: 16,
                 initHeight: 16,
-                blocksInfo: this.host.blocksInfo()
+                blocksInfo: this.host.blocksInfo(),
+                filter: "!tile !dialog !background"
```

Comment:
does this remove these from the gallery entirely? If it removes them completely it would make these inaccessible in text which isn't good (particularly for tile, the others are a bit rarely used)

Reply:
oh yeah you're right. I'll just remove the filter and give it a whirl